.github/workflows/ci.yml

```
@@ -2,42 +2,46 @@
+        
+    - name: Install frontend dependencies
+      run: npm ci
+      
+    - name: Install backend dependencies
+      run: |
+        cd backend
+        npm ci
+        
+    - name: Type check frontend
+      run: npm run type-check
+      
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)